INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py

```diff
@@ -44,19 +46,19 @@ def SERIAL_SINGLE_MEASUREMENT(dc_inputs, params):
     # The first reading is commonly empty.
     s = ser.readline().decode()
 
-    # Some readings may be empty. Try a second time if so.
+    # Some readings may be empty at first because it takes time before receiving the data. Try a second time if so.
```

Comment:
How about doing it with a while loop as follows:
```
s = ""
while s == "":
        s = ser.readline().decode()
 reading = s[:-2].split(",")
 // rest of the code
```

Reply:
Yes definitely !!